src/py/run_generator.rs

```diff
@@ -98,17 +98,14 @@ fn convert_spend(
 
     let mut new_coins = Vec::<PyConditionWithArgs>::new();
     for new_coin in spend_cond.create_coin {
-        let mut cwa = PyConditionWithArgs {
+        new_coins.push(PyConditionWithArgs {
             opcode: CREATE_COIN,
             vars: vec![
                 PyBytes::new(py, &new_coin.puzzle_hash).into(),
                 int_to_pybytes(py, new_coin.amount),
+                node_to_pybytes(py, a, new_coin.hint),
             ],
-        };
-        if new_coin.hint != a.null() {
```

Comment:
This would only work for some nulls anyway, since a clvm empty string `""` would not match.